src/js/components/Video.js

```diff
@@ -106,26 +125,60 @@ export default class Video extends Component {
         <RefreshIcon size={controlIconSize} /> :
           <PlayIcon size={controlIconSize} />));
 
+    var fullScreenButton;
+    if (this.props.allowFullScreen) {
+      fullScreenButton = (
+        <Button plain={true} onClick={this._onFullScreen}>
+          <ExpandIcon />
+        </Button>
+      );
+    }
+
+    var videoHeader = (
+      <Box />
+    );
+
+    if (this.props.videoHeader) {
+      videoHeader = this.props.videoHeader;
+    } else if (fullScreenButton) {
+      // fallback to only displaying full screen icon in header
+      // if allowing fullscreen
+      videoHeader = (
+        <Box align="end" full="horizontal">
+          {fullScreenButton}
+        </Box>
+      );
+    }
+
     var title;
     if (this.props.title) {
       classes.push(`${CLASS_ROOT}--titled`);
       title = (
-        <div className={`${CLASS_ROOT}__title`}>
+        <Box align="center" justify="center" className={`${CLASS_ROOT}__title`}>
```

Comment:
I think we might still want the ${CLASS_ROOT}__title if we want to be able to hide the title when the video is playing and being interacted with (i.e., just show a pause button on hover, per estories design). This might be only an estories design.